tracer/src/Datadog.Trace.SourceGenerators/AspectsDefinitions/AspectsDefinitionsGenerator.cs

```diff
@@ -218,7 +220,7 @@ private static string GetFullName(ISymbol? member)
     {
         if (member != null && member is IMethodSymbol methodSymbol)
         {
-            string arguments = string.Join(",", methodSymbol.Parameters.Select(a => GetFullName(a.Type)));
+            string arguments = string.Join(",", methodSymbol.Parameters.Select(a => GetFullName(a.Type, false)));
```

Comment:
nit: Don't use Linq in source generators because perf. (not for this PR, for your refactor!)

Reply:
Noted 👍